extensions/datastores/kudu/src/main/java/org/locationtech/geowave/datastore/kudu/operations/KuduRangeRead.java

```diff
@@ -119,21 +124,24 @@ protected KuduRangeRead(
 
             KuduScannerBuilder scannerBuilder = operations.getScannerBuilder(table);
             KuduScanner scanner =
-                scannerBuilder.addPredicate(lowerPred).addPredicate(upperPred).addPredicate(
-                    partitionPred).addPredicate(adapterIdPred).build();
+                scannerBuilder.addPredicate(adapterIdPred).addPredicate(lowerPred).addPredicate(
```

Comment:
Why is the ordering of the predicates changed?

Reply:
It follows the declaration order, so it makes it easier to read and verify.